setuptools/__init__.py

```diff
@@ -224,5 +224,25 @@ def findall(dir=os.curdir):
     return list(files)
 
 
+def get_version(path, field='__version__'):
+    """
+    Get version information from a text file. Version is extracted
+    from "key = value" format with key matched at the beginning of
+    a line and specified by `field` parameter. Returns string.
+    """
+    version_file = os.path.abspath(path)
```

Comment:
I suspect this line has no useful effect.

Reply:
Could be. Just a safe practice, useful for debugging.